### DISCUSSION TITLE
新增聊天

### DISCUSSION THREAD

**Discussion by YangZonghang** [Meloong-Git/PCL#3894] [提问] [answered]
聊天，增强PCL的交互功能，游戏新增 PCLchat MOD，用户自定义添加

**Comment by MoYuan-CN** [comment, marked as answer]
啊？
PCL 是 **Minecraft 启动器**

**Reply by YangZonghang** [reply]
为PCL增强交互功能，交互问题是网易版玩家不喜欢玩Java、国际基岩版的跟被原因，原版的聊天界面交互功能太差

**Reply by allMagicNB** [reply]
> 为PCL增强交互功能，交互问题是网易版玩家不喜欢玩Java、国际基岩版的跟被原因，原版的聊天界面交互功能太差

想得美哦，不考虑下服务器会被攻击？

**Reply by allMagicNB** [reply]
> > > 为PCL增强交互功能，交互问题是网易版玩家不喜欢玩Java、国际基岩版的跟被原因，原版的聊天界面交互功能太差
> > 
> > 想得美哦，不考虑下服务器会被攻击？
> 
> 你好好阅读一下 discussion 正文。他的意思是**新增一个聊天 mod 的快速安装**

聊天 Mod 不是 PCL 的？

**Comment by allMagicNB** [comment]
不是哥你要干啥？

**Comment by LTCatt** [comment]
不要交各种做梦的东西，以及请在发帖前先搜索是否有重复项。这是一次提醒，继续瞎水可能会被禁言。

**Comment by Deep-Dark-Forest** [comment]
> 游戏新增 PCLchat MOD，用户自定义添加

https://github.com/Hex-Dragon/PCL2/issues/2682#issuecomment-1712770092

> 不会通过将常用 Mod 加入自动安装列表的申请，要不然这个列表只会越来越长…… 

这是否意味着即使是 PCL 有服务器也不会这么做（）
